runtests.py

```diff
@@ -0,0 +1,4 @@
+from __future__ import unicode_literals
+from mezzanine.bin.runtests import main
+
+main('drum')
```

Comment:
This script should be referenced from setup.py, see an example here: https://github.com/stephenmcd/cartridge/blob/c6d2465075ffd2fccc2766d51c81d05b6e86cb3c/setup.py#L52-L53